Assets/Scripts/Internal/DaggerfallInterior.cs

```diff
@@ -620,7 +620,7 @@ private void AddFlats(PlayerGPS.DiscoveredBuilding buildingData)
                 GameObject go = GameObjectHelper.CreateDaggerfallBillboardGameObject(obj.TextureArchive, obj.TextureRecord, node.transform);
 
                 // Set position
-                DaggerfallBillboard dfBillboard = go.GetComponent<DaggerfallBillboard>();
+                Billboard dfBillboard = go.GetComponent<Billboard>();
```

Comment:
For consistency, I think this part of the code should run on custom billboards too (currently, the loop stops at line 617 for them). 
he custom flat gameobjects were designed for 3d models in mind, but we've found use for it with custom billboard classes too. So I think as long as it has a `Billboard` component, it should have its elevation adjusted by DFU in all cases